examples/whitelabel/README.md

```diff
@@ -6,14 +6,44 @@
 
 ### Instructions
 
-Copy the files from the `assets` folder (next to this README file) to your Botpress Server installation's `assets` folder. You should end up having a structure similar to the following:
+Copy the files from the `assets` folder (next to this README file) to your Botpress Server installation's `data/assets` folder. You should end up having a structure similar to the following:
 
 ```
-├── assets
-│   ├── custom
-│   │   └── logo.png
-│   └── custom-theme.css
+├── data
+├──── assets
```

Comment:
Shouldn't this line look like:
│   └── assets
To reflect that it is a child folder to data?